backend/src/database/data.js

```diff
@@ -0,0 +1,22 @@
+export const medicationRequestToRemsAdmins = Object.freeze([
+  {
+    rxnorm: 2183126,
+    display: 'Turalio 200 MG Oral Capsule',
+    remsAdminFhirUrl: process.env.REMS_ADMIN_FHIR_URL || 'http://localhost:8090/4_0_0'
```

Comment:
this env var is used for the docker-compose-dev.yml, docker-compose.yml, and docker-compose-local-build.yml in rems-setup